No rule to make target `dev-builder-image'

Checklist:

* [x] I've searched in the docs and FAQ for my answer: http://bit.ly/argocd-faq.
* [x] I've included steps to reproduce the bug.
* [x] I've pasted the output of `argocd version`.

**Describe the bug**

No rule to make target `dev-builder-image' in Makefile.
```
make dev-builder-image
make: *** No rule to make target `dev-builder-image'.  Stop.
```

**To Reproduce**

```
git clone https://github.com/argoproj/argo-cd.git
cd argo-cd
dep ensure
make dev-builder-image
```

**Expected behavior**

execute `make dev-builder-image` successfully.

**Screenshots**

If applicable, add screenshots to help explain your problem.

**Version**

```shell
argocd version
argocd: v1.2.5+85f62df
  BuildDate: 2019-10-29T00:07:08Z
  GitCommit: 85f62dff9e3de5a2ee80e9350c897923f59b7e85
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: darwin/amd64
FATA[0000] token contains an invalid number of segments argocd version
argocd: v1.2.5+85f62df
  BuildDate: 2019-10-29T00:07:08Z
  GitCommit: 85f62dff9e3de5a2ee80e9350c897923f59b7e85
  GitTreeState: clean
  GoVersion: go1.12.6
  Compiler: gc
  Platform: darwin/amd64
FATA[0000] token contains an invalid number of segments 
```

**Logs**

```
```

--- a/docs/CONTRIBUTING.md
+++ b/docs/CONTRIBUTING.md
@@ -55,7 +55,7 @@ Ensure dependencies are up to date first:
 
 ```shell
 dep ensure
-make dev-builder-image
+make dev-tools-image
 make install-lint-tools
 ```
 

